src/pybind/mgr/dashboard/frontend/src/app/shared/api/settings.service.ts

```diff
@@ -14,6 +22,14 @@ export class SettingsService {
 
   private settings: { [url: string]: string } = {};
 
+  getValue(name: string) {
```

Comment:
Could you please add a tests for this method?

Reply:
Done